Classes/Tree/TcaTreeSelectorWidget.php

```diff
@@ -142,11 +142,11 @@ public function renderTcaTreeSelectorWidget(array $parameters= [], TYPO3\CMS\Bac
     /**
      * Initialize widget settings and variables
      *
-     * @param array $parameters Parameters passed by TCA rendering call
-     * @param t3lib_TCEforms $fobj t3lib_TCEforms object passed by TCA rendering call
+     * @param array $parameter Parameters passed by TCA rendering call
+     * @param \TYPO3\CMS\Backend\Form\Element\AbstractFormElement $fobj t3lib_TCEforms object passed by TCA rendering call
      */
-    protected function init($parameters = [], TYPO3\CMS\Backend\Form\Element\AbstractFormElement $fobj = null) {
-        parent::init($parameters, $fobj);
+    protected function init(array $parameter = [], TYPO3\CMS\Backend\Form\Element\AbstractFormElement $fobj = null) {
```

Comment:
Wenn die Variable im AbstractTcaWidget.php $params heisst, dann muss die doch auch hier so heissen?

Reply:
Ich dachte, es ist nur eine Name, wusste nicht, dass compiler da probleme machen könnte.

Reply:
Anyways: Die Namen wurden zu den anderen Variablennamen in der Klasse angepasst